ballerina-tests/tests/48_bidi_timestamp_service.bal

```diff
@@ -34,20 +34,20 @@ service "BidiStreamingTimestampService" on ep48 {
     }
 
     remote function bidiStreamingGreetBoth(BidiStreamingTimestampServiceBiDiGreetingCaller caller,
-     stream<BiDiGreeting, grpc:Error?> clientStream) returns error? {
-        check clientStream.forEach(function(BiDiGreeting value) {
+    stream<BiDiGreeting, grpc:Error?> clientStream) {
+        checkpanic clientStream.forEach(function(BiDiGreeting value) {
             BiDiGreeting greeting = {
                 name: value.name,
                 time: checkpanic time:utcFromString("2012-12-03T11:13:30.472Z")
             };
             checkpanic caller->sendBiDiGreeting(greeting);
         });
     }
-    
+
     remote function bidiStreamingExchangeTime(BidiStreamingTimestampServiceTimestampCaller caller,
-     stream<time:Utc, grpc:Error?> clientStream) returns error? {
-        time:Utc responseTime = check time:utcFromString("2021-12-03T11:13:30.472Z");
-        check clientStream.forEach(function(time:Utc value) {
+    stream<time:Utc, grpc:Error?> clientStream) {
+        time:Utc responseTime = checkpanic time:utcFromString("2021-12-03T11:13:30.472Z");
+        checkpanic clientStream.forEach(function(time:Utc value) {
```

Comment:
```suggestion
    stream<time:Utc, grpc:Error?> clientStream) returns error? {
        time:Utc responseTime = check time:utcFromString("2021-12-03T11:13:30.472Z");
        check clientStream.forEach(function(time:Utc value) {
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/606/commits/c32ef7e98e2bfaa4df966a96025d16d755dfb1cd